XROLE/storeManus_v2
# Ch react setup #162778340

#### What does this pull request do?
- Install React and React-dom
#### Description of task to be completed?
-[x] Installed React
-[x] Installed React-dom
#### How should this be manually tested?
- Clone the repo
- open the package.json file, you will see the React and React-dom in it 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
- [#162778340]
